### PR TITLE
20 waiting for ranking view

### PR DESCRIPTION
**Pull request by nac5504**
#20

### CODE REVIEW

File: Guess-E/WaitingForRankingView.swift [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                                        timer.upstream.connect().cancel()
+                                        self.status = "Time is up!"
+                                    } else if(self.status.hasSuffix("...")) {
+                                        self.status.removeLast(2)
+                                        
+                                    } else {
+                                        self.status += "."
+                                    }
+
+                                            }
```
Comment (atlant04) [inline Guess-E/WaitingForRankingView.swift:122]: nit: can you fix the alignment for } and throughout the document for easy of reading. I think if you select everything and do ctl + I, it should do it

**Review by atlant04 — COMMENTED** [review]
looks good, see comments for some nits

File: Guess-E/WaitingForRankingView.swift [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                            .frame(height: 250)
+                            .cornerRadius(30)
+                    }.padding()
+                    
+                }.frame(width: 394, alignment: .leading)
+                Spacer()
+                
+                    ZStack(alignment: .leading){
+                        RoundedRectangle(cornerRadius: 10)
+                            .fill(Color(#colorLiteral(red: 0, green: 0, blue: 0, alpha: 0.11999999731779099)))
```
[the remark this answers is not in the document]
Comment (nac5504) [reply Guess-E/WaitingForRankingView.swift:102]: Ok ill fix it. I didnt actually write that I copied it from someone else's branch lol

**Comment by nac5504** [comment]
fixed all of the errors

**Review by atlant04 — APPROVED** [review]

**Pull request merged**